code/game/objects/effects/projectile/projectile_tracer.dm

```diff
@@ -0,0 +1,62 @@
+/proc/generate_tracer_between_points(datum/point/starting, datum/point/ending, beam_type, color, qdel_in = 5)		//Do not pass z-crossing points as that will not be properly (and likely will never be properly until it's absolutely needed) supported!
```

Comment:
Is this condition ever checked for? Either here or in the upper level?

Reply:
what condition?

Reply:
fixed.